NEWS.md

```diff
@@ -27,6 +27,8 @@
   more informative warnings when supplied with set aesthetics
   (i.e., `slope`, `intercept`, `yintercept`, and/or `xintercept`)
   and mapped aesthetics (i.e., `data` and/or `mapping`).
+  
+* `stat_density2d()` can now take an `adjust` parameter to scale the default bandwidth. (#2860, @haleyjeppson)
```

Comment:
For future reference, the best place to put new bullets is at the top.

Reply:
Ahh okay, noted!